Failure to compile

**Describe the bug**
Failure to compile. Message:

Error: node_modules/ang-jsoneditor/jsoneditor/jsoneditor.component.d.ts:57:87 - error TS2344: Type '{ options: { alias: "options"; required: false; }; debug: { alias: "debug"; required: false; }; data: { alias: "data"; required: false; }; }' does not satisfy the constraint '{ [key: string]: string; }'.
  Property '"options"' is incompatible with index signature.
    Type '{ alias: "options"; required: false; }' is not assignable to type 'string'.

57     static ɵcmp: i0.ɵɵComponentDeclaration<JsonEditorComponent, "json-editor", never, { "options": { "alias": "options"; "required": false; }; "debug": { "alias": "debug"; "required": false; }; "data": { "alias": "data"; "required": false; }; }, { "change": "change"; "jsonChange": "jsonChange"; }, never, never, false, never>;

**To Reproduce**
Serve application after installation.

**To Fix**
Changed the last line in the jsoneditor.component.d.ts file to:
    static ɵcmp: i0.ɵɵComponentDeclaration<JsonEditorComponent, "json-editor", never, 
    { 
        "options": "options" ; 
         "debug": "debug"  ; 
         "data": "data"  }, 
    { 
        "change": "change"; 
        "jsonChange": "jsonChange"; }, 
        never, never, false, never>;
}

**Expected behavior**
No errors

**Console and Error stack trace**
If applicable, add screenshots to help explain your problem.
Use debug mode to see in your console the data and options passed to jsoneditor. Copy this and paste in your issue when reporting bugs. Copy the stacktrace as well!

Error: src/app/admin/vertex-json/vertex-json.component.html:30:5 - error NG8002: Can't bind to 'options' since it isn't a known property of 'json-editor'


**Additional context**
Matched all configurations to the working StackBlitz, but didn't work.

--- a/README.md
+++ b/README.md
@@ -7,7 +7,7 @@ Angular Json Editor (wrapper for [jsoneditor](https://github.com/josdejong/jsone
 
 **[Live Demo](https://mariohmol.github.io/ang-jsoneditor/)** | [StackBlitz template](https://stackblitz.com/edit/ang-jsoneditor)
 
-Working with Angular 17 / 18 / 19 / 20.
+Working with Angular 17 / 18 / 19 / 20 / 21. Requires Angular 17 or higher.
 
 ![Demo Image](/src/assets/printDemo.png)
 

